src/main/scala/wdlTools/util/Util.scala

```diff
@@ -193,36 +195,44 @@ object Util {
     *  trimmed, as well as `w` whitespace characters trimmed from the
     *  beginning of each line.
     *  @example
-    *    val s = "   \n  hello\n   goodbye  "
-    *    stripLeadingWhitespace(s) => (1, 2, "hello\n goodbye")
+    *    val s = "   \n  hello\n   goodbye\n "
+    *    stripLeadingWhitespace(s, false) => (1, 1, "hello\n  goodbye\n")
+    *     stripLeadingWhitespace(s, true) => (1, 2, "hello\n goodbye")
     */
-  def stripLeadingWhitespace(s: String): (Int, Int, String) = {
-    val lines = s.split(System.lineSeparator())
-    val wsRegex = "^\\s*$".r
-    val nonWsRegex = "^(\\s.*)(.*)$".r
-    val (lineOffset, content) = lines.foldLeft((0, Vector.empty[(Int, String)])) {
-      case (l, wsRegex(_)) if l._2.isEmpty => (l._1 + 1, l._2)
-      case (l, nonWsRegex(ws, txt))        => (l._1, l._2 :+ (ws.length, txt))
-      case other                           => throw new Exception(s"Unexpected non-matching line ${other}")
+  def stripLeadingWhitespace(s: String,
+                             ignoreEmptyLast: Boolean = false,
+                             lineSep: String = System.lineSeparator()): (Int, Int, String) = {
+    val lines = s.split("\r\n?|\n")
+    val wsRegex = "^([ \t]*)$".r
+    val nonWsRegex = "^([ \t]*)(.+)$".r
+    val (lineOffset, content) = lines.foldLeft((0, Vector.empty[(String, String)])) {
+      case ((lineOffset, content), wsRegex(_)) if content.isEmpty => (lineOffset + 1, content)
+      case ((lineOffset, content), nonWsRegex(ws, txt))           => (lineOffset, content :+ (ws, txt))
+      case ((lineOffset, content), txt)                           => (lineOffset, content :+ ("", txt))
     }
     if (content.isEmpty) {
       (lineOffset, 0, "")
     } else {
-      val (wsLengths, strippedLines) = content.unzip
-      val colOffset = wsLengths.min
-      (lineOffset,
-       colOffset,
-       (
-           if (colOffset == 0) {
-             strippedLines
-           } else {
-             strippedLines.zip(wsLengths).map {
-               case (line, wsLength) if wsLength > colOffset =>
-                 (" " * (wsLength - colOffset)) + line
-               case (line, _) => line
-             }
-           }
-       ).mkString(System.lineSeparator()))
+      val (whitespace, strippedLines) = (
+          if (ignoreEmptyLast && content.nonEmpty && content.last._2.trim.isEmpty) {
+            content.slice(0, content.length - 1)
```

Comment:
I think you can just do `content.tail` here

Reply:
tail is everything after the first item. I'm trying to get everything except the last item.